File: Assignment23/Program1.c
// 1.Write a program which accept string from user and convert it
// into lower case.
// Input :   "Marvellous Multi OS"
// Output :   marvellous multi os

#include <stdio.h>
void strlwrx(char *str)
{
    while (*str != '\0')
    {
        if (*str >= 'A' && *str <= 'Z')
        {
            *str = *str + 32;
            printf("%c", *str);
        }
        else
        {
            printf("%c", *str);
        }
        str++;
    }
}
int main()
{
    char Arr[20];

    printf("Enter string\n");
    scanf("%[^'\n']s", &Arr);

    strlwrx(Arr);
    return 0;
}

File: Assignment31/Program1.java
// 1.Write a Java program which accept string from user and count number of capital characters. 
// Input :   "Marvellous Multi OS"
// Output :   4 

import java.util.*;

class Marvellous extends StringX
{
	public int CountCapital()
	{
		char Arr[] = str.toCharArray();
		int iCnt = 0,i = 0;
		for(i = 0;i < Arr.length;i++)
		{
			if((Arr[i] >= 'A') && (Arr[i] <= 'Z'))
			{
				iCnt++;
			}
		}
		return iCnt;
	}
}

class StringX
{
	public String str;

	public void Accept()
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter String");
		str = sobj.nextLine(); 
	}
	public void Display()
	{
		System.out.println("String is : "+str);
	}
}

class Program1
{
	public static void main(String[] args) 
	{
		Marvellous mobj = new Marvellous();
		int iRet = 0;
		mobj.Accept();
		mobj.Display();
		iRet = mobj.CountCapital();
		System.out.println("Number of Capital letters are :"+iRet);
	}
}
